canonical-websites/www.ubuntu.com
# Remove Unity 8 preview from /desktop/ubuntu-kylin

## Done

Remove Unity 8 preview from /desktop/ubuntu-kylin

## QA

- Pull code and run ./run
- Go to: http://localhost:8001/desktop/ubuntu-kylin
- Check that the Unity 8 preview section has gone a per copy doc

### Design QA

Go to https://www.ubuntu.com/desktop/ubuntu-kylin and note the final section 'Unity 8 preview’ then pop over to http://www.ubuntu.com-1704-desktop-unity.demo.haus/desktop/ubuntu-kylin and not that the section has been removed 

## Issue / Card

Card: https://trello.com/c/5421cMFB
Doc: https://docs.google.com/a/canonical.com/document/d/1eJVoek0vT_vT4vy1Quw0NaBqm1fR9K2Dw8korIT2pKc/edit?disco=AAAABIz1RwE
